BatchExamples/DFIRBatch.reb

```diff
@@ -2693,6 +2693,181 @@ Keys:
+        Category: Third Party Applications
+        KeyPath: CurrentControlSet\Services\RManService
+        Recursive: true
+        Comment: "Displays artifacts relating to RemoteUtilities"
+    -
+        Description: RemoteUtilities
+        HiveType: SYSTEM
+        Category: Third Party Applications
+        KeyPath: Usoris\Remote Utilities\RManService\Host\Parameters
+        Recursive: true
+        Comment: "Displays artifacts relating to Portable RemoteUtilities Configuration"
+    -
+        Description: RemoteUtilities
+        HiveType: NTUSER
+        Category: Third Party Applications
+        KeyPath: Software\Usoris\Remote Utilities\RManService\Host\Parameters
+        ValueName: General
+        Recursive: false
+        Comment: "Displays artifacts relating to RemoteUtilities Configuration - Base64 decode output"
+    -
+        Description: RemoteUtilities
+        HiveType: NTUSER
+        Category: Third Party Applications
+        KeyPath: Software\Usoris\Remote Utilities\RManService\Host\Parameters
+        ValueName: InternetId
+        Recursive: false
+        Comment: "Displays artifacts relating to RemoteUtilities Configuration - Base64 decode output"
+    -
+        Description: RemoteUtilities
+        HiveType: NTUSER
+        Category: Third Party Applications
+        KeyPath: Software\Usoris\Remote Utilities\RManService\Host\Parameters
+        ValueName: Security
+        Recursive: false
+        Comment: "Displays artifacts relating to RemoteUtilities Configuration - Base64 decode output"
+    -
+        Description: RemoteUtilities
+        HiveType: NTUSER
+        Category: Third Party Applications
+        KeyPath: Software\Usoris\Remote Utilities\RManService\Host\Parameters
+        ValueName: FUSClientPath
+        Recursive: false
+        Comment: "Displays artifacts relating to RemoteUtilities Configuration"       
+# https://jsac.jpcert.or.jp/archive/2023/pdf/JSAC2023_1_1_yamashige-nakatani-tanaka_en.pdf
+
+# Third Party Applications -> Splashtop - https://www.splashtop.com/
+
+    -
+        Description: Splashtop
+        HiveType: SYSTEM
+        Category: Third Party Applications
+        KeyPath: CurrentControlSet\Services\SplashtopRemoteService
+        Recursive: true
+        Comment: "Displays artifacts relating to Splashtop"
+    -
+        Description: Splashtop
+        HiveType: SYSTEM
+        Category: Third Party Applications
+        KeyPath: CurrentControlSet\Services\SSUService
+        Recursive: true
+        Comment: "Displays artifacts relating to Splashtop"
+# https://jsac.jpcert.or.jp/archive/2023/pdf/JSAC2023_1_1_yamashige-nakatani-tanaka_en.pdf
+
+# Third Party Applications -> TeamViewer - https://www.teamviewer.com/en-us/
+
+    -
+        Description: TeamViewer
+        HiveType: SYSTEM
+        Category: Third Party Applications
+        KeyPath: CurrentControlSet\Services\TeamViewer
+        Recursive: true
+        Comment: "Displays artifacts relating to Splashtop"
```

Comment:
This should say TeamViewer

Reply:
@reece394 good catch, did you want to do a PR to correct this?

Reply:
Currently on my phone without access to my computer so don’t want to cause issues with the file. If it still isn’t corrected tomorrow I will create a pull request fixing it

Reply:
No worries! I can do the same later tonight too. Not a rush as the artifacts simply won't parse if they don't exist. Everything valid will still work!

Reply:
https://github.com/AndrewRathbun/RECmd/commit/ab5a21d1a8603329a01e59b36cc1c82ea0b582e8